oak_marketplace_emotion_recognition/args_parser.py

```diff
@@ -0,0 +1,30 @@
+from argparse import ArgumentParser
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "-model",
+        "--root_model_path",
+        help="Path to root model directory",
+        type=str,
+    )
+    parser.add_argument(
+        "-vis",
+        "--visualization",
+        action="store_true",
+        help="Visualize the results from the network",
+    )
+    parser.add_argument(
+        "--output-video",
+        "-out_vid",
+        help="Save by-frame visualization results of the inference into video",
+        action="store_true",
+    )
```

Comment:
Let's simplify this a little bit so it would be easier for users to run 

```python
parser.add_argument(
    "--save-video",
    type=int,
    choices=[0, 1],
    help="If set to 1, save the visualization results onto a MP4 file"
    default=0,
)
```